src/components/Header.tsx

```diff
@@ -6,22 +6,22 @@ import axios, { AxiosError } from 'axios'
 import { ErrorResponse } from '../types'
 import { useIsLoggedIn, useLogout } from '../utils/hooks'
 import { getTokenLocalStorage } from '../utils/helpers'
-import { ErrorContext, ErrorType } from '../context/error'
 import { ProfileContext } from '../context/profile'
 import { TokenContext } from '../context/token'
 import { getProfile } from '../services/api'
 import { BACKEND_URL } from '../services/constants'
 import useTranslationHook from '../hook/TranslationHook'
+import { MessageContext, MessageType } from '../context/message'
 
 const Header = (): JSX.Element => {
   const { token, setToken } = useContext(TokenContext)
   const { profile, setProfile } = useContext(ProfileContext)
-  const { addError } = useContext(ErrorContext)
+  const { addMessage } = useContext(MessageContext)
   const logout = useLogout()
   const { t } = useTranslationHook()
 
   const addErrorCallback = useCallback(
-    (error: ErrorType) => addError(error),
+    (error: MessageType) => addMessage(error),
```

Comment:
Let's keep the FlashMessage wording so it doesn't get mixed up with other type of messaging we could get in the future